docker-compose.yml

```diff
@@ -27,6 +27,10 @@ services:
       PORT: 4000
       NEO4J_USER: "neo4j"
       NEO4J_PASSWORD: "letmein"
+      JWT_SECRET: "not-so-secret"
+      JWT_AUTH_KEYS: "[{\"id\":\"local\",\"apiKey\":\"PZsG+oEW3K3QOoB5z0f30InzjXdBqM9LMtJa7BTg1xo=\",\"scopes\":[\"*\"]}]"
```

Comment:
Maybe to avoid having to escape this json correctly, we could use configs or secrets: https://docs.docker.com/engine/swarm/configs/#defining-and-using-configs-in-compose-files

Reply:
I would like to keep this compatible with the current tools we are using to deploy the ce-api and those only use env variables. The `JWT_AUTH_KEYS ` is not something that would need to be modified a lot and for online environments only by us. Changing a key is just a simple replace in the text.